.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @saradrada @garzuzo @holgiosalos @danielgalvis98 @leonleo997 @manuelq12
+* @holgiosalos @renardete @Valeryibarra 
```

Comment:
this change is not needed, please review the updated documentation
https://github.com/renardete/Cypress-Training-Doc